packages/gestalt/src/SearchField.js

```diff
@@ -108,6 +108,7 @@ const SearchField = ({
             },
           }}
           left
+          right
```

Comment:
How does this work?

https://developer.mozilla.org/en-US/docs/Web/CSS/position

![image](https://user-images.githubusercontent.com/127199/86802888-51b1fd00-c02a-11ea-8c08-05b8a4d5bc65.png)